lib/config.py

```diff
@@ -117,8 +117,9 @@
              default='host-os.ks'),
     ('--host-os-packages-groups',):
         dict(help='Packages groups in yum repository'),
-    ('--automated-install-packages-groups',):
-        dict(help='Packages and packages groups which are installed using automated installation', nargs='*'),
+    ('--iso-root-fs-packages-groups',):
+        dict(help='Packages and packages groups that will be installed in '
+             'the ISO root file system.', nargs='*'),
```

Comment:
These packages and packages groups belong to the base distro (CentOS), it seems? It'd be good to mention it somewhere.